lib/AppInfo/Application.php

```diff
@@ -60,6 +64,11 @@ public function __construct (array $urlParams = array()) {
 		});
 		$container->registerCapability('Capabilities');
 
+		$container->getServer()->getEventDispatcher()->addListener('user.afterdelete', function (GenericEvent $event) use ($container) {
```

Comment:
It might be worthy to move this block to another function, similarly to what we have for the `setupConsumerAndNotifier` function. Note that you'll need to call the function in the app.php file.